pcdsdevices/valve.py

```diff
@@ -858,3 +858,40 @@ class VCN_VAT590(BaseInterface, Device):
     )
 
     status = Cpt(VCN_VAT590_Status, '',)
+
+
+class VFV(Device):
+    """
+    VFV = Variable Frequency Valves
+    """
```

Comment:
Do you know precisely which struct this corresponds to in the vacuum lib? I think it's helpful to include this sort of info here for understanding/clarity purposes (and for review)

Like how the docstring for VCN says:
```
It corresponds to ST_VCN in the lcls-twincat-vacuum library.
```